<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="sidebarFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="toggleSidebarButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navHomeButton">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navMonitorButton">
         <property name="text">
          <string>Monitor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navTransmitButton">
         <property name="text">
          <string>Transmit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navDashboardButton">
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navSettingsButton">
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navHelpButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="homePage">
       <widget class="QLabel" name="titleWelcome">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>60</y>
          <width>501</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Welcome to CAN Emulator Tool</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="subtitleWelcome">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>140</y>
          <width>431</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Monitor • Send • Visualize CAN frames</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>250</y>
          <width>91</width>
          <height>67</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="connectButton">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disconnectButton">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>470</y>
          <width>481</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="navMonitorButton_2">
           <property name="text">
            <string>Start monitor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="navTransmitButton_2">
           <property name="text">
            <string>Send test frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="navDashboardButton_2">
           <property name="text">
            <string>View dashboard</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>280</y>
          <width>291</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="labelComPort"/>
         </item>
         <item>
          <widget class="QComboBox" name="labelBaud">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="monitorPage"/>
      <widget class="QWidget" name="transmitPage"/>
      <widget class="QWidget" name="dashboardPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
